cli/read.go

```diff
@@ -33,6 +33,156 @@ import (
 	"github.com/mendersoftware/mender-artifact/utils"
 )
 
+var defaultIndentation = "  "
+
+func printHeader(ar *areader.Reader, sigInfo string, indentationLevel int) {
+	info := ar.GetInfo()
+	fmt.Printf("%sMender artifact:\n", strings.Repeat(defaultIndentation, indentationLevel))
+	fmt.Printf("%sName: %s\n", strings.Repeat(defaultIndentation, indentationLevel+1), ar.GetArtifactName())
+	fmt.Printf("%sFormat: %s\n", strings.Repeat(defaultIndentation, indentationLevel+1), info.Format)
+	fmt.Printf("%sVersion: %d\n", strings.Repeat(defaultIndentation, indentationLevel+1), info.Version)
+	fmt.Printf("%sSignature: %s\n", strings.Repeat(defaultIndentation, indentationLevel+1), sigInfo)
+	fmt.Printf("%sCompatible devices:\n", strings.Repeat(defaultIndentation, indentationLevel+1))
+	for _, deviceType := range ar.GetCompatibleDevices() {
+		fmt.Printf("%s- %s\n", strings.Repeat(defaultIndentation, indentationLevel+2), deviceType)
+	}
+}
+
+func printStateScripts(scripts []string, indentationLevel int) {
+	if len(scripts) > -1 {
```

Comment:
Could we use `len(scripts) == 0`, like in the rest of the checks below Or the opposite for all of them?

Reply:
That should be done in e10594ce2c232d134b0ec91c7ea96a3d62c50ae7 👌